l10n_es_pos/static/src/js/models.js

```diff
@@ -66,28 +83,33 @@ odoo.define("l10n_es_pos.models", function (require) {
             var total = order_super.get_total_with_tax.apply(this, arguments);
             var below_limit = total <= this.pos.config.l10n_es_simplified_invoice_limit;
             this.is_simplified_invoice =
-                below_limit && this.pos.config.iface_l10n_es_simplified_invoice;
+                below_limit && this.pos.config.is_simplified_config;
             return total;
         },
         set_simple_inv_number: function () {
-            const self = this;
             return this.pos
                 .get_simple_inv_next_number()
-                .then(function (configs) {
-                    const config = configs[0];
-                    self.pos.config.l10n_es_simplified_invoice_number =
-                        config.l10n_es_simplified_invoice_number;
-                    const simplified_invoice_number =
-                        self.pos.config.l10n_es_simplified_invoice_prefix +
-                        self.pos.get_padding_simple_inv(
-                            config.l10n_es_simplified_invoice_number
-                        );
-                    self.l10n_es_unique_id = simplified_invoice_number;
-                    self.is_simplified_invoice = true;
+                .then(([config]) => {
+                    // We'll get the number from DB only when we're online. Otherwise
+                    // the sequence will run on the client side until the orders are
+                    // synced.
+                    this.pos._set_simplified_invoice_number(config);
                 })
-                .catch(function () {
-                    self.l10n_es_unique_id = self.uid;
-                    self.is_simplified_invoice = true;
+                .catch((error) => {
+                    // We'll only consider an error if a code is returned for
+                    // the error and doesn't contain 'pending_orders'
+                    if (
+                        error.message.code &&
+                        !error.message.code.includes("pending_orders")
+                    ) {
+                        throw error;
+                    }
+                })
```

Comment:
He realizado unos cambios en la forma de detectar si se trata de un error de sincronización o de otro tipo, ya que en la V15 ha cambiado el error que se devuelve. 

A ver si lo podéis revisar @pedrobaeza @chienandalu